zerver/forms.py

```diff
@@ -208,6 +216,20 @@ def clean_email(self) -> str:
         if realm.is_zephyr_mirror_realm:
             email_is_not_mit_mailing_list(email)
 
+        if settings.BILLING_ENABLED:
+            try:
+                check_license_limit(realm)
+            except LicenseLimitError:
+                message = f"Signup of {email} failed because of license limit. Please increase the licenses."
```

Comment:
Where and to whom is the above message shown? I want to make sure I have the right context before taking an edit pass. :)

Reply:
This is sent to the signup notifications private stream. 

![Screenshot from 2021-05-25 16-34-20](https://user-images.githubusercontent.com/7190633/119487517-1ec64400-bd77-11eb-8f3e-5f408718d055.png)


In the screenshot, you can see that the messages changed slightly since I tweaked them a bit now.